File: 백준/Bronze/10807. 개수 세기/개수 세기.py
'''
1. 아이디어
- N개의 주어진 정수 중, 구하려는 정수 v가 몇 번 존재하는지 개수를 print
2. 시간복잡도
- N번 정수 확인하기: O(N)
3. 자료구조
- int N,V
- int[] num_list

'''
import sys
input = sys.stdin.readline

N = int(input())
num_list = list(map(int,input().split()))
V = int(input())
cnt = 0

for i in num_list:
    if i == V:
        cnt += 1
        
print(cnt)

File: 백준/Bronze/10807. 개수 세기/개수 세기.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        int N = scanner.nextInt();
        scanner.nextLine();  // ← 개행 문자 제거
        
        String[] numList = scanner.nextLine().split(" ");
        
        String v = scanner.next(); // 찾고 싶은 값 입력
        
        int cnt = 0;
        
        for (int i = 0; i < N; i++) {
            if (numList[i].equals(v)) {
                cnt++;
            }
        }
        
        System.out.println(cnt);
    }
}
